### PR TITLE
feat(digifinex): add spot <> swap to transfer

### PR DESCRIPTION
**Pull request by yzh-pelle**

### CODE REVIEW

File: ts/src/digifinex.ts [hunk 1]
```diff
@@ -2908,12 +2909,23 @@ export default class digifinex extends Exchange {
```
Comment (carlosmiei) [inline ts/src/digifinex.ts:2909]: @yzh-pelle can you please add a static request test validating the fix?

Comment (yzh-pelle) [reply ts/src/digifinex.ts:2909]: Yeah, I'll do it

File: ts/src/digifinex.ts [hunk 2]
```diff
@@ -2952,31 +2965,57 @@ export default class digifinex extends Exchange {
...
-        //     }
-        //
+        const request = {};
+        const fromSwap = (fromAccount === 'swap');
+        const toSwap = (toAccount === 'swap');
+        let response = undefined;
+        const amountString = this.currencyToPrecision (code, amount);
+        if (fromSwap || toSwap) {
+            if ((fromId !== '1') && (toId !== '1')) {
+                throw new ExchangeError (this.id + ' transfer() supports transferring from and to swap account between spot and swap only');
```
Comment (carlosmiei) [inline ts/src/digifinex.ts:2991]: @yzh-pelle This error message is a bit confusing 😅

File: ts/src/test/static/request/digifinex.json [hunk 3]
```diff
@@ -714,6 +714,32 @@
                   "USDT"
                 ]
             }
+        ],
+        "transfer": [
```
Comment (carlosmiei) [inline ts/src/test/static/request/digifinex.json:718]: @yzh-pelle can you add a test here, transferring from swap to spot?

Comment (yzh-pelle) [reply ts/src/test/static/request/digifinex.json:718]: done

File: ts/src/digifinex.ts (hunk 2 above)
Comment (yzh-pelle) [reply ts/src/digifinex.ts:2991]: @carlosmiei 
I don't know how to word it better 😅
Any ideas?

Comment (carlosmiei) [reply ts/src/digifinex.ts:2991]: @yzh-pelle  Maybe just a few commas missing and minor adjustments, like 

```
transfer() supports transferring from, spot->swap, spot->margin and swap->spot only.
```

Comment (yzh-pelle) [reply ts/src/digifinex.ts]:
```
transfer() supports transferring between spot and swap, spot and margin, spot and OTC only
```
Is it good?

Comment (carlosmiei) [reply ts/src/digifinex.ts]: @yzh-pelle yess

**Pull request merged**